rust/core-lib/src/rpc.rs

```diff
@@ -43,6 +43,8 @@ pub enum Request<'a> {
 #[derive(Debug, PartialEq, Eq)]
 pub enum CoreCommand<'a> {
     Edit { view_id: &'a str, edit_command: EditCommand<'a> },
+    /// A command from the client to a plugin.
+    Plugin {  plugin_command: PluginCommand },
```

Comment:
I wonder if we should have the `plugin_name` here rather than `PluginCommand`, so each variant doesn't need to have it.